tests/vite-app/cypress.config.ts

```diff
@@ -5,9 +5,21 @@ export default defineConfig({
     // We've imported your old cypress plugins here.
     // You may want to clean this up later by importing these.
     setupNodeEvents(on, config) {
+      // https://docs.cypress.io/api/node-events/browser-launch-api
+      on("before:browser:launch", (browser, launchOptions) => {
+        if (browser.family === "chromium" && browser.name !== "electron") {
+          // https://developer.mozilla.org/en-US/docs/Web/API/EcdsaParams
+          // From version 113: Ed25519 algorithm is behind the #enable-experimental-web-platform-features preference
+          launchOptions.args.push(
+            "--enable-experimental-web-platform-features"
+          );
+
+          return launchOptions;
+        }
+      });
       // eslint-disable-next-line @typescript-eslint/no-var-requires
       return require("./cypress/plugins/index.js")(on, config);
     },
-    baseUrl: "http://localhost:5173/",
+    baseUrl: "http://127.0.0.1:5173/",
```

Comment:
On my mac, I get Cypress errors with the `localhost` to `127.0.0.1` change, both when I run headless for tests

```
Cypress could not verify that this server is running:

  > http://127.0.0.1:5173/

We are verifying this server because it has been configured as your baseUrl.
```

or I open the UI (so I can test in Chrome and Edge)

```
Warning: Cannot Connect Base Url Warning

Cypress could not verify that this server is running:

http://127.0.0.1:4173/

This server has been configured as your baseUrl, and tests will likely fail if it is not running.
```

Reply:
this resolves with a `vite.config.ts` change